### ISSUE TITLE
Switching DNS From One Pantheon Site to Another Doc Update 

### ISSUE THREAD

**Issue by ari-gold** [pantheon-systems/documentation#3579]
Re: [Switching DNS From One Pantheon Site to Another](https://pantheon.io/docs/switching-dns/)

Priority (Low‚ Medium‚ High): High

## Issue Description:

Verify and document this method works for uninterrupted HTTPS
(1) Leave the DNS as-is for the moment.
(2) Remove the domain from the old site's Pantheon dashboard. Immediately add it to the new one.
(3) Wait for HTTPS to provision on the new site.
(4) Update DNS

and remove https://pantheon.io/docs/switching-dns/#determine-the-new-dns-records and notice that a 1hr maintenance window is required

**Comment by rachelwhitton** [comment]
@ari-gold what about sites redirecting to `www`? Should the user run through this process start-finish two times? OR should they go through it once - applying each step to both domains?

**Comment by rachelwhitton** [comment]
Can you clarify issue resolution here? I think I'm on the same page, but want to be sure. 

The request is to replace the entire section **HTTPS on the New Site**, including the subsection titled ** Determine the New DNS Records** -- replacing it with the process identified in your description. With the expectation that HTTPS will be uninterrupted when switching a domain from one dashboard to another. 

Yeah? Thanks! Doing some tests now

**Comment by rachelwhitton** [comment]
Test went pretty smooth.. I did get this response at one point: 
![screen shot 2018-04-17 at 3 34 04 pm](https://user-images.githubusercontent.com/10119525/38895311-57f44056-4255-11e8-9558-d5c94e75a0d6.png)

It was a brief state, couple of seconds at best. On refresh (no force) the new site loaded as expected. Tested went from an existing WP site to a new D8 site, then back again - only saw the above once. 

Planning to document it as a single process that applies each step to both domains
